### PR TITLE
Add keras highway

### PR DESCRIPTION
**Pull request by SeaOfOcean**
## What changes were proposed in this pull request?

Add keras highway

## How was this patch tested?

unit test

### CODE REVIEW

**Comment by SeaOfOcean** [comment]
pr validate pass

**Comment by SeaOfOcean** [comment]
pr validate pass

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Highway.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  /**
+   * Densely connected highway network.
+   * Highway layers are a natural extension of LSTMs to feedforward networks.
+   *
+   * @param size input size
+   * @param withBias whether to include a bias
+   * @param activation name of activation function to use
+   * @tparam T Numeric type. Only support float/double now
+   */
+  def apply[@specialized(Float, Double) T: ClassTag](size: Int, withBias: Boolean = true,
```
Comment (hkvision) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Highway.scala]: Can we support weight regularizers?

**Comment by SeaOfOcean** [comment]
pr validate pass

**Pull request merged**